Alignment/OfflineValidation/bin/Zmumumerge.cc

```diff
@@ -0,0 +1,310 @@
+  c->cd();
+  gStyle->SetOptStat(0);
+  th1d_input[0]->SetTitle("");
+  for (int idx_file = 0; idx_file < file_number; idx_file++) {
+    th1d_input[idx_file]->SetMarkerColor(colors_array[idx_file]);
+    th1d_input[idx_file]->SetLineColor(colors_array[idx_file]);
+    th1d_input[idx_file]->Draw("same");
+    lg->AddEntry(th1d_input[idx_file], label_names[idx_file]);
+  }
+  lg->Draw("same");
+  c->SaveAs(output_name);
+}
+
+int Zmumumerge(int argc, char* argv[]) {
+  vector<TString> vec_single_file_path;
+  vector<TString> vec_single_file_name;
+  vector<TString> vec_global_tag;
+  vec_single_file_path.clear();
+  vec_single_file_name.clear();
+  vec_global_tag.clear();
+  Options options;
+  options.helper(argc, argv);
+  options.parser(argc, argv);
+  pt::ptree main_tree;
+  pt::read_json(options.config, main_tree);
+  pt::ptree alignments = main_tree.get_child("alignments");
+  pt::ptree validation = main_tree.get_child("validation");
+  for (const std::pair<std::string, pt::ptree>& childTree : alignments) {
+    vec_single_file_path.push_back(childTree.second.get<std::string>("file"));
+    vec_single_file_name.push_back(childTree.second.get<std::string>("file") + "/Zmumu.root");
+    vec_global_tag.push_back(childTree.second.get<std::string>("globaltag"));
+    //Fitting_GetMassmeanVSvariables(childTree.second.get<std::string>("file") + "/Zmumu.root", childTree.second.get<std::string>("file"));
+  }
+  TString merge_output = main_tree.get<std::string>("output");
+  //=============================================
+  vector<TString> vec_single_fittingoutput;
+  vec_single_fittingoutput.clear();
+  for (unsigned i = 0; i < vec_single_file_path.size(); i++) {
+    Fitting_GetMassmeanVSvariables(vec_single_file_name[i], vec_single_file_path[i]);
+    vec_single_fittingoutput.push_back(vec_single_file_path[i] + "/fitting_output.root");
+  }
+
+  int files_number = vec_single_file_path.size();
+  cout << "files_number=" << files_number << endl;
+  for (int idx_variable = 0; idx_variable < variables_number; idx_variable++) {
+    TString th1d_name = Form("th1d_meanmass_%s", tstring_variables_name[idx_variable].Data());
+    Draw_TH1D_forMultiRootFiles(
+        files_number,
+        vec_single_fittingoutput,
+        vec_global_tag,
+        th1d_name,
+        merge_output + Form("/meanmass_%s_GTs.pdf", tstring_variables_name[idx_variable].Data()));
+    TString th1d_name_entries = Form("th1d_entries_%s", tstring_variables_name[idx_variable].Data());
+    Draw_TH1D_forMultiRootFiles(
+        files_number,
+        vec_single_fittingoutput,
+        vec_global_tag,
+        th1d_name_entries,
+        merge_output + Form("/entries_%s_GTs.pdf", tstring_variables_name[idx_variable].Data()));
+  }
+  //=============================================
+  return EXIT_SUCCESS;
+}
+#ifndef DOXYGEN_SHOULD_SKIP_THIS
+int main(int argc, char* argv[]) {
+  return Zmumumerge(argc, argv);
+  //return Zmumumerge(argc, argv);

```

Comment:
?

Reply:
//return Zmumumerge(argc, argv); 
This is deleted.